medusa/init/__init__.py

```diff
@@ -11,6 +11,10 @@
 import site
 import sys
 
+import six
+
+from .. import app
```

Comment:
Please do not use relative imports, the project uses `from medusa import app`.

Reply:
done